### PR TITLE
[TR] behat : fix label search

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                  | A
| ------------------ | ---
| Migration script?  |n
| Behats             |runnin
| Specs & Static     |y
| Changelog updated  |n

### CODE REVIEW

File: features/Context/NavigationContext.php [hunk 1]
```diff
@@ -672,7 +672,6 @@ public function openPage($page, array $options = [])
             if ($this->loginIfRequired()) {
                 $page = $this->getCurrentPage()->open($options);
                 $this->wait();
-                echo 'login was required' . PHP_EOL;
```
Comment (jmleroux) [inline features/Context/NavigationContext.php]: not needed anymore

**Comment by phaseinducer** [comment]
GTM :+1:

**Comment by skeleton** [comment]
gtm

**Comment by grena** [comment]
GTM

**Pull request merged**